### PR TITLE
feat: Add eip1559 to be displayed when user is setting fee for dapp i…

### PR DESCRIPTION
**Pull request by AndonMitev**
feat: Add eip1559 to be displayed when user is setting fee for dapp interaction

# :books: [Linear Ticket](https://linear.app/liquality/issue/LIQ-970/gas-is-displaying-as-nan-and-network-fee-as-object-object-when-a) :books:

### CODE REVIEW

**Review by beemi — APPROVED** [review]

File: src/components/CustomFeesEIP1559.vue [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <NavBar :showBackButton="false" :showBack="false" :showMenuList="false">
       Network Speed/Fee
     </NavBar>
-    <div class="wrapper form">
+    <div class="wrapper form" :class="{ wrapperPromiseSend: promiseSendView }">
```
Comment (monokh) [inline src/components/CustomFeesEIP1559.vue:6]: `promiseSendView`
that seems like a very weird name when what it does it add some padding. Can't you call it something more relevant? even sometihng like `padLabels`

I don't know the specifics but it doesn't seem appropriate for this component to look different based on where its embedded...

File: src/views/PermissionSend.vue [hunk 2]
```diff
@@ -271,7 +289,12 @@ export default {
     },
     applyCustomFee({ fee }) {
       const presetFee = Object.entries(this.assetFees).find(
-        ([speed, speedFee]) => speed !== 'custom' && speedFee.fee === fee
+        ([speed, speedFee]) =>
+          speed !== 'custom' &&
```
Comment (monokh) [inline src/views/PermissionSend.vue:293]: These few lines are really hard to follow. anyway to improve it?

you can name statements for example

const feeMatches = speedFee.fee.maxPriorityFeePerGas === fee.maxPriorityFeePerGas

**Review by monokh — CHANGES_REQUESTED** [review]

File: src/views/PermissionSend.vue (hunk 2 above)
Comment (AndonMitev) [reply src/views/PermissionSend.vue]: Fixed, splited to 2 varialbes: `isEIP1559Fee` and `isLegacyFee` for better readability

File: src/components/CustomFeesEIP1559.vue (hunk 1 above)
Comment (AndonMitev) [reply src/components/CustomFeesEIP1559.vue]: applied mate
